README.md

```diff
@@ -61,9 +61,9 @@ NOTE: These challenges are currently BLOCKED until Osmosis adds support for IBC
 
 | Task                 | Description                                                                                                                              | Reward         | PR                                                       | Claimed by |
 | :------------------- | :--------------------------------------------------------------------------------------------------------------------------------------- | :------------- | -------------------------------------------------------- | :--------- |
-| Query price over IBC | Query price information for a trading pair over IBC for use in a contract.                                                               | Partial Payout | [#5](https://github.com/CosmosContracts/bounties/pull/5) | @giansalex |
```

Comment:
just to clarify that I have not received a partial payment here.

Reply:
> 

Can I see your initial contract code

> A rule has been defined and the challenges can only be completed, when deployed on mainnet , @JakeHartnell or has this changed?
> 
> My initial contracts were deployed on feb 25th
> 
> query price https://blueprints.juno.giansalex.dev/#/transactions/9785A116F7A10C3E759039D10CAC0C6D84AF943867953CE3FA321A68390D1532
> 
> swap, liquidity https://blueprints.juno.giansalex.dev/#/transactions/1D92F49E1E6FA2ECD291AADE6E0997A1E0441BA6025FEDE518F80148C1688465
> 
> I have been working on this for quite some time, initially with a native [ibc module as suggested by Jake](https://github.com/CosmosContracts/bounties/pull/5#issuecomment-1040825560), and later with contracts because osmosis supports it. [Ibc module](https://github.com/disperze/ibc-osmo)
> 
> I have been checking all the necessary steps to get the osmos proposal approved, making a fully functional contract for anyone. I will publish a draft of the proposal today or tomorrow
> 
> ** regardless of who wins the challenges, i think everyone should receive rewards from Junohacks.

Can I see your original code
also, why you don't publish your code

Reply:
> just to clarify that I have not received a partial payment here.
okay, I'll change the pr

Reply:
this is the contract used at that time https://github.com/giansalex/cw-osmo-price , but using a native module on the osmosis side https://github.com/disperze/ibc-osmo/tree/old-intergamm/x/intergamm